utils/github_api.py

```diff
@@ -0,0 +1,33 @@
+import os
+from pathlib import Path
+from urllib.parse import urlparse
+
+import github
+
+from utils import secret_reader
+
+
+GH_BASE_URL = os.environ.get('GITHUB_API', 'https://api.github.com')
+
+
+class GithubApi:
```

Comment:
A small comment describing what this class is for would be helpful (although I see it we don't do it a lot in qontract-reconcile classes 😂 )

Reply:
sure!